sdcm/results_analyze/test.py

```diff
@@ -538,10 +544,15 @@ def get_same_tests_query(self):
 
     def get_prior_tests(self, filter_path=None) -> typing.List['TestResultClass']:
         output = []
-        es_query = self.get_same_tests_query()
-        es_result = ES().search(index=self._es_data['_index'], q=es_query, filter_path=filter_path,
-                                size=10000)  # pylint: disable=unexpected-keyword-arg
-        es_result = es_result.get('hits', {}).get('hits', None) if es_result else None
+        try:
+            es_query = self.get_same_tests_query()
+            es_result = ES().search(index=self._es_data['_index'], q=es_query, filter_path=filter_path,
+                                    size=10000)  # pylint: disable=unexpected-keyword-arg
+            es_result = es_result.get('hits', {}).get('hits', None) if es_result else None
+        except Exception as exc:  # pylint: disable=broad-except
+            LOGGER.warning("Unable to find ES data: {exc}".format(exc=exc))
```

Comment:
use lazy interpolation

Reply:
done

Reply:
same here, don't see the change

Reply:
corrected for real this time, somehow missed both